docs/source/fx.rst

```diff
@@ -818,11 +818,7 @@ symbolic tracing:
 
         fx.symbolic_trace(f) # Fails!
 
-        def wrapper(flag):
-            return lambda x: f(x, flag)
-
-        new_f = wrapper(flag=True)
-        fx.symbolic_trace(new_f)
+        fx.symbolic_trace(f, concrete_args={'flag': True})
```

Comment:
I'd rework the lead-in sentence above the code example to mention something like "or by binding concrete values to arguments during symbolic tracing"